app/assets/javascripts/components/locales/ja.jsx

```diff
@@ -23,25 +24,24 @@ const ja = {
   "column.mutes": "ミュートしたユーザー",
   "column.notifications": "通知",
   "column.public": "連合タイムライン",
-  "column_back_button.label": "戻る",
   "compose_form.placeholder": "今なにしてる？",
-  "compose_form.privacy_disclaimer": "あなたの非公開カツは返信先のユーザー（at {domains}）に公開されます。{domainsCount, plural, one {that server} other {those servers}}を信頼しますか？投稿のプライバシー保護はKirakiratterサーバー内でのみ有効です。 もし{domains} {domainsCount, plural, one {is not a Mastodon instance} other {are not Mastodon instances}}ならばあなたの投稿のプライバシーは保護されず、ブーストされたり予期しないユーザーに見られる可能性があります。",
+  "compose_form.privacy_disclaimer": "あなたの非公開カツは返信先のユーザー（at {domains}）に公開されます。{domainsCount, plural, one {that server} other {those servers}}を信頼しますか？投稿のプライバシー保護はMastodonサーバー内でのみ有効です。 もし{domains} {domainsCount, plural, one {is not a Mastodon instance} other {are not Mastodon instances}}ならばあなたの投稿のプライバシーは保護されず、ブーストされたり予期しないユーザーに見られる可能性があります。",
```

Comment:
This originally said "Kirakiratter" but when looking at the full message in the English localization, it's clear that it should be Mastodon (because it's talking about Mastodon instances in general).